README.md

```diff
@@ -1,2 +1,45 @@
 # hexgrid-react
 A library to render objects using hexagonal grid layout.
+
+# :arrow_down:Important
+
+  
+Follow these steps:hammer: for contributing:
+
+:green_heart:Fork the repo.
+
+:pushpin:Create a separate branch for any bug/feature with branch name: <grid/bug>
+
+:white_check_mark:Commit once a program is complete.
+
+:pencil2:Fetch for any changes in the original repo by creating an upstream using:
+
+>git remote add upstream https://github.com/nishihere19/hexgrid-react.git
+
+>git fetch upstream
+
+:tada:Merge the changes in your branch.
+
+>git merge upstream/(your_branch_name)
+
+
+:alien:Push your changes to your branch.
+
+:wrench:Make a pull request from your branch.
+
+:heavy_plus_sign:Before submitting a PR, check your code well and explain your work properly in the PR description.
+
+  
+
+# :bookmark:Code advice
+
+Kindly maintain a clean code with proper comments.
+
+Commit messages should be written carefully and describe the changes made.
+
+Create separate PR for a single program update/bug fix.
```

Comment:
Change program update to 'feature'.

Reply:
Also add "Happy contributing" in the end.

Reply:
okay